mypy/plugins/dataclasses.py

```diff
@@ -83,7 +83,13 @@ def transform(self) -> None:
             # Some definitions are not ready, defer() should be already called.
             return
         for attr in attributes:
-            if info[attr.name].type is None:
+            try:
+                node = info[attr.name]
+            except KeyError:
```

Comment:
I think that it would be slightly cleaner if you'd use a `if info.get(attr.name) is None` check instead of try/except here.

Reply:
Fixed.